provider/lpwindow/compute_task.go

```diff
@@ -74,10 +74,10 @@ type WdPostTask struct {
 }
 
 type wdTaskIdentity struct {
-	Sp_id                uint64
-	Proving_period_start abi.ChainEpoch
-	Deadline_index       uint64
-	Partition_index      uint64
+	SpID               uint64
+	ProvingPeriodStart abi.ChainEpoch
+	DeadlineIndex      uint64
+	PartitionIndex     uint64
```

Comment:
This goes into db.Select, I'm not sure if pgx will handle this correctly (does it only care about order?).

Reply:
It does a matching game where it replaces underscores with capitals and id --> ID.
but when in doubt, you can use struct tags:
`db:"sp_id"`